packages/@lwc/integration-karma/test/light-dom/slot-fowarding/slots/forwarding/index.spec.js

```diff
@@ -0,0 +1,133 @@
+import { createElement } from 'lwc';
+import { extractDataIds } from 'test-utils';
+
+import LightContainer from './x/lightContainer/lightContainer';
+
+const vFragBookend = process.env.API_VERSION > 59 ? document.createComment('') : '';
+const commentNode = document.createComment(' comments being forwarded ');
+
+describe('slot forwarding', () => {
```

Comment:
These tests asserts that forwarding slots between light and shadow DOM should produce the same results.